### ISSUE TITLE
MaskFeat training support ?

### ISSUE THREAD

**Issue by BIGBALLON** [facebookresearch/SlowFast#530]
thanks for the great project, I just want to know is there any plan to release MaskFeat pre-trained model or MViT training code. :smile_cat:  

> Our results on standard video benchmarks are groundbreaking: MaskFeat pre-trained MViT-L [56] gets 86.7%
top-1 accuracy on Kinetics-400 [51] without using any external data, greatly surpassing the best prior number of this
kind by +5.2%, and also methods using large-scale image
datasets, e.g., IN-21K and JFT-300M [80]. When transferring to downstream tasks, MaskFeat gets unprecedented
results of 38.8 mAP on action detection (AVA [42]) and
75.0% top-1 accuracy on human-object interaction classification (SSv2 [40]). When generalized to the image domain,
MaskFeat also obtains competitive 84.0% top-1 with ViT-B
and 85.7% with ViT-L using only ImageNet-1K [24].
Our code will be available in PyTorchVideo1,2[29, 30].

**Comment by adeobootpin** [comment]
I agree, great project. Eagerly waiting for MaskFeat training code. Please give us a taste soon! We are salivating! :)
